internal/services/controller/controller.go

```diff
@@ -9,6 +9,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	policyv1 "k8s.io/api/policy/v1"
```

Comment:
hmm is linter not enabled for our agent? Imports should be ordered and grouped into three groups:
vanilla
3rd party
our

Reply:
will take a look into our linter